components/FileTable.tsx

```diff
@@ -230,14 +242,12 @@ const FileDownloadModal: React.FunctionComponent<IFileDownloadModalProps> = (
         doesFileIncludeLevel1OrLevel2SequencingData
     );
     const synapseFiles = props.files.filter(
-        (f) =>
-            f.downloadSource === "Synapse"
+        (f) => f.downloadSource === 'Synapse'
     );
-    const lowerLevelImagingFiles = props.files.filter((f) =>
-        f.Component.startsWith('Imaging') && (
-            f.level === "Level 1" ||
-            f.level == "Level 2"
-        )
+    const lowerLevelImagingFiles = props.files.filter(
+        (f) =>
+            f.Component.startsWith('Imaging') &&
+            (f.level === 'Level 1' || f.level == 'Level 2')
```

Comment:
this is a prettier change. there is actually no new code change here. not sure why it didn't format this segment before.